components/browser/awesomebar/src/main/java/mozilla/components/browser/awesomebar/layout/DefaultSuggestionViewHolder.kt

```diff
@@ -42,6 +43,10 @@ internal sealed class DefaultSuggestionViewHolder {
             } else {
                 descriptionView.visibility = View.VISIBLE
                 descriptionView.text = suggestion.description
+
+                val titleParams = titleView.layoutParams as MarginLayoutParams
+                titleParams.setMargins(0, 0, 0, 0)
+                titleView.requestLayout()
```

Comment:
I'm not really sure whys this part is needed. Doesn't that remove all margin? 🤔

Reply:
Apologies, I come from the web world -- is there a better way to only modify the top margin?

Reply:
You mean something like this? https://stackoverflow.com/a/50765990

Reply:
@darkwing Only modifying top margin is not directly possible but you can set the value of the other items to the currently assigned value (e.g. `setMargins(leftMargin, [..])`). Or we set it explicitly to 8dp like in the XML?

What's important here is that the views get recycled (or reused) if they are no longer needed, which means we  need to make sure that the if/else cases are symmetric and we potentially reset all values. So we should make sure to set the margin in both cases!